test/test_nn.py

```diff
@@ -5087,6 +5087,21 @@ def test_upsamplingBilinear2d(self):
                 input = torch.randn(1, 1, 2, 2, requires_grad=True)
                 gradcheck(lambda x: F.upsample(x, out_size, **kwargs), [input])
 
+    def test_upsamplingBicubic2d(self):
+        for align_corners in [True, False]:
+            kwargs = dict(mode='bicubic', align_corners=align_corners)
+
+            # test float scale factor up & downsampling
+            for scale_factor in [0.5, 1.5, 2]:
+                in_t = torch.ones(1, 1, 2, 2)
```

Comment:
Can you add some tests for correctness, e.g., comparing output values?

Moreover, can you add module test in the giant list below?